src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -100,22 +106,14 @@ protected unsafe IUnknownVftbl VftblIUnknown
             }
         }
 
-        private protected unsafe IUnknownVftbl VftblIUnknownFromOriginalContext
-        {
-            get
-            {
-                ThrowIfDisposed();
-                return **(IUnknownVftbl**)ThisPtrFromOriginalContext;
-            }
-        }
-
-        protected IObjectReference(IntPtr thisPtr)
+        protected IObjectReference(IntPtr thisPtr, IntPtr contextPtr)
```

Comment:
This is both a source and a binary breaking change. Is this intended? If so, does this mean that we have accepted the idea of making breaking changes in the next CsWinRT release? Because if so, does that mean we can then also go fix more issues that are elsewhere in the library that would also require breaking changes?

Reply:
I don't think the constructors are meant to be called from outside WinRT.Runtime? I believe that for the nativeaot work we are already making breaking changes and this would go as part of that set of changes. /cc @manodasanW

Reply:
If the breaking change is contained within `WinRT.Runtime` uses, it is acceptable for the staging branch.  In this case, will projections generated using the new `FactoryObjectReference` added in the other PR break?  If so, we might want to keep the existing protected constructor with one parameter and add the two parameter one as a new one.

Reply:
Added it back.